### PR TITLE
Semantic parsing intro chapter

### PR DESCRIPTION
**Pull request by pdasigi**

### CODE REVIEW

**Comment by matt-gardner** [comment]
Sorry, did I drop the ball on this?  I never got a review request, I just noticed that you submitted #148 which depends on this one.

**Comment by pdasigi** [comment]
@matt-gardner I requested your review for an earlier PR from my branch to `semantic_parsing`. This one is for releasing the chapter by merging into `master`, and I wanted to do it at a less busy time than end-of-2020. I think now is a good time. Any concerns?

File: chapters/overview.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ prior knowledge of AllenNLP—the whole point of this guide is to provide onboar
 
 ## Part 3
 
+* [Semantic parsing: Intro and Seq2Seq model](/semantic-parsing-seq2seq)
+* [Semantic parsing: Grammar-based decoding](/semantic-parsing-grammar)
```
Comment (matt-gardner) [inline chapters/overview.md:97]: Some of these should be labeled as "Coming soon", right?

File: chapters/part3/semantic-parsing-grammar.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (matt-gardner) [inline chapters/part3/semantic-parsing-grammar.md:1]: Probably better to keep these unfinished ones in the branch.

**Review by matt-gardner — APPROVED** [review]
I'm trusting my former self and not looking at the chapter content again =).  The unfinished content should be removed from this PR, but otherwise LGTM!

File: chapters/overview.md (hunk 1 above)
Comment (pdasigi) [reply chapters/overview.md]: Yes. I indicated that these chapters are coming soon in the "Coming Soon" chapter. :)

File: chapters/part3/semantic-parsing-grammar.md (hunk 2 above)
Comment (pdasigi) [reply chapters/part3/semantic-parsing-grammar.md]: Done.

**Pull request merged**